Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -0,0 +1,31 @@
+- type: entity
+  parent: DrinkGlass
+  id: DrinkLPPMiruvorGlass
+  suffix: miruvor
+  components:
+  - type: SolutionContainerManager
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)